tests/repository_simulator.py

```diff
@@ -116,6 +124,8 @@ def __init__(self) -> None:
         self.dump_dir: Optional[str] = None
         self.dump_version = 0
 
+        self.fetch_tracker: FetchCounter = FetchCounter()
```

Comment:
```suggestion
        self.fetch_tracker = FetchCounter()
```
I think `mypy` and the code editors are smart enough to infer the type in this case.